src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            const url = article.html.match(/<a.*?href\s*?=\s*?["'](?!http)(.*?)["'] *?>/)[1];
```

Comment:
Let's just modularize out this regex so it's only written out once. Just make a variable outside the if statement named `absoluteUrlRegex` or something like that, and make it the one with the capture group in it so it'll work for both of them.

After that let's merge this in but I may have another task for you after that.